### DISCUSSION TITLE
Plotting on multiple HDDs

### DISCUSSION THREAD

**Discussion by Putas123** [madMAx43v3r/chia-gigahorse#228] [General]
How to set the parameters of the GH plotter so that the final file is saved to several HDDs successively?
That is:
1 - the finished file is saved on HDD1
2 - before it finishes writing to HDD1, it writes the next file to HDD2
3 - before it finishes writing to HDD1 and HDD2, the next file is being written to HDD3

**Comment by Kruger1981** [comment]
just add -d -d -d

example Windows
-d F:\ -d G:\ -d H:\

example ubuntu
-d /media/server/f/ -d /media/server/g/ -d /media/server/h/

Just replace the HDDs/locations to suite what you have setup on your machine.

if you have any issues you can also reach out on Discord

ha la sempre alguem que ajuda
